core-bundle/contao/drivers/DC_Table.php

```diff
@@ -1300,7 +1300,7 @@ protected function copyChildren($table, $insertID, $id, $parentId)
 							foreach ($GLOBALS['TL_DCA'][$k]['config']['oncopy_callback'] as $callback)
 							{
 								$dc = (new \ReflectionClass(self::class))->newInstanceWithoutConstructor();
-								$dc->table = $k;
+								$dc->strTable = $k;
```

Comment:
Kind of unexpected that this even works with a protected property.

@m-vo @ausi Is this a coincidence or a PHP feature? https://3v4l.org/vZELp